### ISSUE TITLE
Better handling of specific app initialization errors

### ISSUE THREAD

**Issue by radishmouse** [thunderbird/send-suite#85]
Once #66 is merged, continue adding error handling for initialization errors:

- when is it appropriate to send the user back to the moz account login?
- if they've logged in successfully but have no keys, they need to go to the Backup/Recovery page
- if they cannot recover their keys from backup, inform the user that they can no longer access their data

Possibly need a separate ticket for TB Extension because:
- all login is handled through the management page, which is separate from the actual extension
- if the extension detects an error with initialization, find out how to open the management page (or give clear instructions to the user if we can't do that programmatically)
